controller/setting_controller.go

```diff
@@ -251,6 +251,7 @@ func (sc *SettingController) syncBackupTarget() (err error) {
 			// already monitoring
 			return nil
 		}
+		sc.logger.Warn("Restart backup store monitoring")
```

Comment:
```suggestion
		sc.logger.Info("restarting backup store monitor because backup target changed from %v to %v", sc.bsMonitor.backupTarget, targetSetting.Value)
```